tableauserverclient/server/endpoint/views_endpoint.py

```diff
@@ -105,7 +105,7 @@ def csv_fetcher():
     def _get_view_csv(self, view_item, req_options):
         url = "{0}/{1}/data".format(self.baseurl, view_item.id)
 
-        with closing(self.get_request(url, parameters={"stream": True})) as server_response:
+        with closing(self.get_request(url, req_options, {"stream": True})) as server_response:
```

Comment:
Thanks!

This might already by correct, but since I'm not 100% the order of function parameters as they get passed to the underlying `requests.get`, can you make the `{"stream":True}` a named parameter ( `parameters=...`)

Ugh, naming a parameter `parameters` is confusing. Thanks Requests....